NuGetPackages/MonoGame.Framework.iOS.nuspec

```diff
@@ -0,0 +1,24 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/10/nuspec.xsd">
+    <metadata>
+        <id>MonoGame.Framework.iOS</id>
+        <version>3.3.0</version>
+        <title>MonoGame.Framework.iOS</title>
+        <authors>Simon-darkside-Jackson</authors>
```

Comment:
This should be MonoGame Team.  I understand you did all the work on it, but we don't add personal credits to individual features/fixes.